composer.json

```diff
@@ -23,6 +23,7 @@
         "container-interop/container-interop": "^1.1",
         "vlucas/phpdotenv": "^2.2",
         "joomla/registry": "^1.5",
+        "aura/session": "^2.0",
```

Comment:
Don't want to play devil's advocate but why aura and not joomla framework or symphony or... (actually I like aura's packages although I haven't used this one!). This question will come up and will be better if there is a doc with pros/cons and why this was chosen.
The implementation is fine tho since you are using interfaces! 👍